app/config/ConfigDecorator.scala

```diff
@@ -95,7 +88,7 @@ class ConfigDecorator @Inject()(
       .getOrElse("/template/mustache")
 
   def transformUrlForSso(url: URL) =
-    s"$companyAuthFrontendHost/ssoout/non-digital?continue=" + URLEncoder.encode(url.toString, "UTF-8")
+    s"$basGatewayFrontendHost/bas-gateway/ssoout/non-digital?continue=" + URLEncoder.encode(url.toString, "UTF-8")
```

Comment:
@vimalsuresh @anujakhare when logged in as a SA user, going to portal should redirect them to bas-gateway signout first

Reply:
Checked